.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Describe the bug
+
+A clear and concise description of what the bug is.
+
+### Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
looks a little strange for our package

Reply:
@evilebottnawi 

Pardon. For some reason, the changes I made to adapt the GitHub issue template to the repository were not applied. I am trying to recover the changes I made and send you another request for review.

Reply:
Updated: https://github.com/itgalaxy/webfont/pull/206/files#diff-74c2649c999338ed7a2833d79d4b25c3R21-R25